packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -45,6 +45,7 @@ interface RowsListProps {
   timeRange?: TimeRange;
   footerPaginationEnabled: boolean;
   initialRowIndex?: number;
+  expandedRows?: Map<number, React.ReactElement>;
```

Comment:
a bit confusing anming as these are not really expanded (in terms of Table component expanded row)